A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_FuelPredPage.js

```diff
@@ -134,8 +134,8 @@ class CDUFuelPredPage {
                         rteRsvPercentCell = "--.-";
                         rteRSvCellColor = "[color]cyan";
                     } else {
-                        rteRsvWeightCell = mcdu.getRouteReservedWeight().toFixed(1);
-                        rteRsvPercentCell = mcdu.getRouteReservedPercent().toFixed(1);
+                        rteRsvWeightCell = (mcdu.getRouteReservedWeight() * mcdu._conversionWeight).toFixed(1);
+                        rteRsvPercentCell = (mcdu.getRouteReservedPercent() * mcdu._conversionWeight).toFixed(1);
```

Comment:
This is a percentage not a weight

Reply:
I don't understand your suggestion to 138